### PR TITLE
Fix/show two whitelist ips issue 1632

### LINKED ISSUE #1632: [Feature] Add static IP 3.131.104.27 to the list of whitelisted IPs
## Summary

On the Appsmith Cloud, in the "Create Datasource" page, we show a message asking users to whitelist IPs in their db/firewall. We must add `3.131.104.27` to the list of IPs that users must whitelist.

## Motivation
In order to remove any single point of failures in Appsmith Cloud, users must whitelist 2 IPs in their databases/firewalls to ensure continued connectivity with their Appsmith apps.

### PR DESCRIPTION
**Pull request by jsartisan**
Fixes #1632

### CODE REVIEW

File: app/client/src/utils/helpers.tsx [hunk 1]
```diff
@@ -152,3 +152,13 @@ export const isEllipsisActive = (element: HTMLElement | null) => {
...
   );
 };
+
+/**
+ * converts array to sentences
+ * for e.g - ['Pawan', 'Abhinav', 'Hetu'] --> 'Pawan, Abhinav and Hetu'
+ *
+ * @param arr string[]
+ */
+export const toSentence = (arr: string[]) => {
```
Comment (hetunandu) [inline app/client/src/utils/helpers.tsx:162]: Nice util but needs a better function name

File: app/client/cypress/locators/commonlocators.json [hunk 2]
```diff
@@ -76,5 +76,6 @@
   "onPlay": ".t--property-control-onplay .t--open-dropdown-Select-Action",
   "chooseAction": ".single-select",
   "chooseMsgType": ".t--open-dropdown-Select-type",
-  "onPause": ".t--property-control-onpause .t--open-dropdown-Select-Action"
+  "onPause": ".t--property-control-onpause .t--open-dropdown-Select-Action",
+  "collapsibleHelp": ".t--collapsible-help"
```
Comment (hetunandu) [inline app/client/cypress/locators/commonlocators.json:80]: Wonder by this is being added here

**Review by hetunandu — CHANGES_REQUESTED** [review]

File: app/client/cypress/locators/commonlocators.json (hunk 2 above)
Comment (jsartisan) [reply app/client/cypress/locators/commonlocators.json:80]: I was trying to add test cases for the help text. But later on, realized we can't do it.

File: app/client/src/utils/helpers.tsx (hunk 1 above)
Comment (jsartisan) [reply app/client/src/utils/helpers.tsx:162]: I was also thinking the same actually :P 
`convertArrayToSentence` works?

File: app/client/cypress/locators/commonlocators.json (hunk 2 above)
Comment (hetunandu) [reply app/client/cypress/locators/commonlocators.json:80]: Can we remove it now then? That extra class is not relevant in this PR

Comment (jsartisan) [reply app/client/cypress/locators/commonlocators.json]: Done.

**Review by hetunandu — APPROVED** [review]

**Pull request merged**